about.html

```diff
@@ -42,15 +42,15 @@
                     alt="Entrepreneurs summit logo"></a>
```

Comment:
- [ ] Kindly make this logo take us to `index.html` using the `href` attribute 👍